### PR TITLE
Part 2 of pre-release polishing and bugfixes 

### PR DESCRIPTION
**Pull request by math07**
Still a WIP

### CODE REVIEW

File: INSTALL [hunk 1]
```diff
@@ -0,0 +1,34 @@
+There are two ways to install Johnny which depends on your platform.
+1. Use our easy installer (Windows and OS X only)
+OR
```
Comment (shinnok) [inline INSTALL]: Remove "OR"

File: INSTALL [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+2. Download source and build johnny (all platforms)
```
Comment (shinnok) [inline INSTALL]: Let's capitalize Johnny.

File: INSTALL [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+1. Easy-install : use our installer (Windows and OS X only)
+-----------------------------------
+Simply visit http://openwall.info/wiki/john/johnny to get the latest installer
+for your platform. It comes bundled with a working John The Ripper binary.
+So you have everything ready to start cracking passwords.
+
+2. Getting the source code and compiling on Unix/Linux, Windows and OS X
```
Comment (shinnok) [inline INSTALL]: Separate the build instructions per platform family, like in the Tarsnap example I gave you.

File: installer/config/config.xml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (shinnok) [inline installer/config/config.xml]: You should have made the installer work into a separate PR.

File: installer/packages/johnny/data/platforms/ADD_DLLS_HERE.txt [hunk 5]
```diff
@@ -0,0 +1,4 @@
+To make a new installer :
```
Comment (shinnok) [inline installer/packages/johnny/data/platforms/ADD_DLLS_HERE.txt]: Mention that these are Qt platform plugins.

File: installer/packages/johnny/meta/package.xml [hunk 6]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+<Package>
+    <DisplayName>Johnny</DisplayName>
+    <Description>Johnny, the cross-platform GUI, for John The Ripper the popular cracking software.</Description>
```
Comment (shinnok) [inline installer/packages/johnny/meta/package.xml]: A GUI frontend to the John the Ripper password cracker.

File: INSTALL [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+=====================
+
+1. Easy-install : use our installer
+-----------------------------------
+Simply visit http://openwall.info/wiki/john/johnny to get the latest installer
+for your platform and http://openwall.info/wiki/john/custom-builds to get the latest
+John The Ripper binaries.
+
+============================
+Unix/Linux, Windows and OS X
```
Comment (shinnok) [inline INSTALL]: This isn't exactly what I had in mind...

Can you please take another look at the example I wrote for Tarsnap https://github.com/shinnok/tarsnap-gui/blob/master/INSTALL and try a bit harder to come up with something more along the lines of that one?

**Comment by math07** [comment]
Are you okay with the install dir based a lot on tarsnap-gui ?

**Comment by math07** [comment]
I merged this to make v2.0 tag on github. You can still put your comments here regarding the INSTALL file and I'll update it and move the tag to another commit.

**Pull request merged**